### PR TITLE
[4.11] CLOUDSTACK-10299: UI: fix error in network listing in project mode

### PR DESCRIPTION
**Pull request by resmo**
Statically passing account (and domainid) results in a conflict when in project view because projectid and account can not be used together. 

Removing the 2 params. fixes the issue. (some whitespace fixes along)

### CODE REVIEW

File: ui/scripts/network.js [hunk 1]
```diff
@@ -3537,28 +3535,28 @@
...
+
                                                     				vmidipmapIndex++;
-                                                    			}                                                			
+                                                    			}
                                                     		}
-                                                    	}   
-                                                        
+                                                    	}
+
                                                         /*$.ajax({
```
Comment (rafaelweingartner) [inline ui/scripts/network.js]: You have gone over some commented code. What about removing it?

**Comment by rafaelweingartner** [comment]
@resmo thanks for the contributions, but I have a doubt. You say it (the PR) is aimed at 4.11, but you opened a PR against master branch. Which one do you want?

**Comment by resmo** [comment]
@rafaelweingartner changed the base to 4.11

File: ui/scripts/network.js (hunk 1 above)
Comment (resmo) [reply ui/scripts/network.js]: I didn't see a reason why to remove comments, I wanted to fix the issue. Whitespace cleanup was made by my editor automatically.

**Comment by borisstoyanov** [comment]
@blueorangutan package

File: ui/scripts/network.js (hunk 1 above)
Comment (rafaelweingartner) [reply ui/scripts/network.js]: Well.. the commented code is not used. So, there is your reason to remove it. The less "code" (lines) for our eyes to see, the better.

Comment (resmo) [reply ui/scripts/network.js]: I see your point. cleaning up commented out code makes sense.

Comment (rafaelweingartner) [reply ui/scripts/network.js]: Great. Thanks!

File: ui/scripts/network.js [hunk 2]
```diff
@@ -412,8 +412,6 @@
                         $.ajax({
                             url: createURL('listRemoteAccessVpns'),
                             data: {
-                                account: g_account,
```
Comment (rafaelweingartner) [inline ui/scripts/network.js]: Is this call only used in the project view?
I mean, if I list networks and I am out of the project view, should not ACS filter by these data? Or, is it done automatically in the server side when the users is logged-in?

**Comment by resmo** [comment]
@borisstoyanov before or after the patch?

**Comment by borisstoyanov** [comment]
@resmo both if you have. If no, new stuff is what's interesting :)

**Comment by resmo** [comment]
Before

selection_149

After

selection_151

**Comment by resmo** [comment]
ready for review

**Comment by DaanHoogland** [comment]
@borisstoyanov @rafaelweingartner is this merge ready?

**Comment by rafaelweingartner** [comment]
There is still one open question of mine there. It is about some parameters that were removed. I am asking if that removal can affect the loading of information for the default view (view outside of project).

**Comment by resmo** [comment]
@rafaelweingartner I couldn't test the view with having existing remote vpns configured. But my assumption was that the api is working identically as other list apis returning the results of the current owner.

**Comment by rafaelweingartner** [comment]
Got it. I think this one is ready to be merged then.

**Comment by DaanHoogland** [comment]
anything holding you back, @rafaelweingartner ?

**Comment by rafaelweingartner** [comment]
Nothing... I was busy with some other things.
I am merging this one now

**Pull request merged**